README.md

```diff
@@ -60,22 +59,19 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Hello Microverse <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> it the first traial hello microverse project
```

Comment:
- [ ] Great job, @osmanwako! Since this is a message you are trying to pass across, I suggest you do not make it a comment by removing the `>` before the texts for more clarity. This is to ensure that visitors do not miss out on an important message you might want them to grasp :wink: .
- [ ] [OPTIONAL] The project description is too short that it holds little or no descriptive message about your project. Readme's are great opportunities to sell or pitch your design and documentation skills to users or recruiters. Kindly consider making it more elaborate and meaningful. :smile: